test/integration/provider/alicloud.go

```diff
@@ -80,7 +80,7 @@ func getMachines(machineClass *v1alpha1.MachineClass, secretData map[string][]by
 	return machines, nil
 }
 
-// getOrphanesInstances returns list of Orphan resources that couldn't be deleted
+// getOrphanedInstances returns list of Orphan resources that couldn't be deleted.
```

Comment:
it should be 
`getOrphanedInstances returns list of Orphan resources`

Reply:
ok

Reply:
done